source/reference/methods.txt

```diff
@@ -730,6 +730,233 @@ Query Plan Cache Methods
+       primary of the replica set to run this method.
+
+   * - :method:`rs.addArb()`
+
+     - Adds an arbiter to an existing replica set.
+
+   * - :method:`rs.config()`
+
+     - Returns a document that contains the current replica set configuration.
+
+   * - :method:`rs.freeze()`
+
+     - Makes the replica set member that ``mongosh`` is connected to 
+       ineligible to become primary for the specified duration. The 
+       duration must be specified in seconds.
+
+   * - :method:`db.getReplicationInfo()`
+
+     - Returns the status of the replica set from the oplog data.
+
+   * - :method:`rs.initiate()`
+
+     - Initializes a new replica set.
+
+   * - ``rs.isMaster``
+
+     - Returns the replica set configuration, role, and status for 
+       the ``mongod`` instance that ``mongosh`` is connected to. 
+       
+       The ``mongosh`` ``rs.isMaster()`` method wraps the server :dbcommand:`isMaster` command.
+
+   * - :method:`db.printReplicationInfo()`
+
+     - Returns the oplog of the replica set member that ``mongosh`` is 
+       connected to. 
+       
+       This is identical to the :method:`rs.printReplicationInfo()` method.
+
+   * - :method:`rs.printReplicationInfo()`
+
+     - Returns the oplog of the replica set member that ``mongosh`` is 
+       connected to. 
+       
+       This is identical to the :method:`db.printReplicationInfo()` method.
+
+   * - ``db.printSecondaryReplicationInfo``
+
+     - Returns the status of the secondary members of the replica set. 
+     
+       This is identical to the ``rs.printSecondaryReplicationInfo()`` method. 
+       This method's output is similar to the 
+       :method:`db.printSlaveReplicationInfo()` method. The :method:`db.printSlaveReplicationInfo()` method is deprecated in ``mongosh``. Use 
+       ``db.printSecondaryReplicationInfo()`` instead.
+
+       The following is an example output from the 
+       ``rs.printSecondaryReplicationInfo()`` method issued on a replica set with two secondary members:
+
+       .. code-block:: sh 
+          :copyable: false 
+
+          source: rs2.example.net:27017
+          {
+            syncedTo: 'Tue Oct 13 2020  09:37:28 GMT-0700 (Pacific Daylight Time)',
+            replLag: '0 secs (0 hrs) behind the primary '
+          }
+          ---
+          source: rs3.example.net:27017
+          {
+            syncedTo: 'Tue Oct 13 2020  09:37:28 GMT-0700 (Pacific Daylight Time)',
+            replLag: '0 secs (0 hrs) behind the primary '
+          }
+
+   * - ``rs.printSecondaryReplicationInfo``
+
+     - Returns the status of the secondary members of the replica set. 
+     
+       This is identical to the ``db.printSecondaryReplicationInfo()`` 
+       method. This method's output is similar to the 
+       :method:`rs.printSlaveReplicationInfo()` method in the legacy 
+       :binary:`~mongo` shell. The :method:`rs.printSlaveReplicationInfo()` method is deprecated in ``mongosh``. Use 
```

Comment:
if this replacement is configured for this repo.
```suggestion
       |mongo| shell. The :method:`rs.printSlaveReplicationInfo()` method is deprecated in ``mongosh``. Use 
```

Reply:
👍